docs/farming.md

```diff
@@ -31,7 +31,7 @@ You can find these executables in the [Releases](https://github.com/subspace/sub
 
 ```
 # Replace `INSERT_YOUR_ID` with a nickname you choose
-./subspace-node-x86_64-*-snapshot `
+.\subspace-node-x86_64-windows-2019-snapshot-2022-jan-05.exe `
```

Comment:
I don't like this, we'll have to update it all the time. Better write explanation what the name of the file should be.